src/lib/network_pool/transaction_pool.ml

```diff
@@ -601,17 +651,19 @@ struct
             |> Option.is_some )
       in
       if not @@ List.is_empty commit_conflicts_locally_generated then
-        [%log' info t.logger]
-          "Locally generated commands $cmds dropped because they conflicted \
-           with a committed command."
-          ~metadata:
-            [ ( "cmds"
-              , `List
-                  (List.map commit_conflicts_locally_generated
-                     ~f:
-                       Transaction_hash.User_command_with_valid_signature
-                       .to_yojson ) )
-            ] ;
+        List.iter commit_conflicts_locally_generated
```

Comment:
Here you are also only iterating over some of the commit conflicts and not all of them, but my understanding is that the vk table is tracking all txns, not just locally submitted ones.